terraform/application/config/review.tfvars.json

```diff
@@ -1,4 +1,5 @@
 {
+  "app_environment": "review",
```

Comment:
The "config" variable already has this value: https://github.com/DFE-Digital/itt-mentor-services/blob/abf57b2ff9e1af7dd11a7b7e876f22f820da660f/global_config/review.sh#L1
It should be passed to terraform like config_short: https://github.com/DFE-Digital/itt-mentor-services/blob/abf57b2ff9e1af7dd11a7b7e876f22f820da660f/Makefile#L47